cli/constants.py

```diff
@@ -15,6 +17,13 @@
 
 CLI_CONFIG_FILE_NAME = 'remme-core-cli'
 
+FAMILY_NAMES = [
+    RemmeFamilyName.ACCOUNT.value,
+    RemmeFamilyName.NODE_ACCOUNT.value,
+    RemmeFamilyName.PUBLIC_KEY.value,
+    RemmeFamilyName.SWAP.value,
+]
```

Comment:
Does the library have the array of family names or not?

Reply:
@dmytrostriletskyi, latest release https://github.com/Remmeauth/remme-client-python/releases/tag/v1.1.0-alpha has published 3 days ago. 
Now, after updating the dependency in CLI, we have this functionality.